exercises/python/encryption-context-complete/src/document_bucket/api.py

```diff
@@ -0,0 +1,106 @@
+from typing import Dict, Set
+
+import aws_encryption_sdk  # type: ignore
+from aws_encryption_sdk import KMSMasterKeyProvider  # type: ignore
+
+from .model import (ContextItem, ContextQuery, DocumentBundle, PointerItem,
+                    PointerQuery)
+
+
+class DocumentBucketOperations:
```

Comment:
Here and in other places ... are we not going to add docstrings?

Reply:
We are. I left it to a problem for future me for now. I meant to cut an issue, but bikeshedded myself on an issue per language and do I cut all the issues ahead of time or ... 

https://github.com/aws-samples/busy-engineers-document-bucket/issues/23